cmd/clusterctl/internal/test/fake_objects.go

```diff
@@ -59,6 +60,16 @@ func NewFakeCluster(namespace, name string) *FakeCluster {
 	}
 }
 
+// NewFakeClusterWithForceLabel is same to NewFakeCluster except it will also return a secret
+// named <clustername>-cloud-config with clusterctl.cluster.x-k8s.io/move label
+func NewFakeClusterWithForceLabel(namespace, name string) *FakeCluster {
+	return &FakeCluster{
+		namespace:      namespace,
+		name:           name,
+		withUserSecret: true,
```

Comment:
```suggestion
		withCloudConfigSecret: true,
```
to better refrect the use case we are testing

Reply:
sure